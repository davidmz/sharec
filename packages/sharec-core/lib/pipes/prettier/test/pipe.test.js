const { fixtures } = require('testUtils')
const { pipe } = require('../pipe')

describe('strategies > pipes > prettier > pipe', () => {
  describe('JSON', () => {
    const perttierFxt = fixtures('prettier/json/00-base')

    it('should merge configs', () => {
      expect(pipe('.prettierrc')(perttierFxt)).toMatchSnapshot()
    })
  })

  describe('YAML', () => {
    const perttierFxt = fixtures('prettier/yaml/00-base')

    it('should merge configs', () => {
      expect(pipe('.prettierrc.yaml')(perttierFxt)).toMatchSnapshot()
    })
  })
})
