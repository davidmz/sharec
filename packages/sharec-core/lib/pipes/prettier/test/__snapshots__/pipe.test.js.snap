// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`strategies > pipes > prettier > pipe JSON should merge configs 1`] = `
"{
  \\"tab\\": true,
  \\"tabWidth\\": 2,
  \\"semi\\": true
}"
`;

exports[`strategies > pipes > prettier > pipe YAML should merge configs 1`] = `
"tab: true
tabWidth: 2
semi: true"
`;
